devicetypes/smartthings/zigbee-motion-detector.src/zigbee-motion-detector.groovy

```diff
@@ -67,11 +68,20 @@ def installed(){
 def parse(String description) {
 	log.debug "description(): $description"
 	def map = zigbee.getEvent(description)
-	if(!map){
+	ZoneStatus zs
+	if (!map) {
 		if (description?.startsWith('zone status')) {
-			map = parseIasMessage(description)
+			zs = zigbee.parseZoneStatus(description)
+			map = parseIasMessage(zs)
 		} else {
-			map = batteyHandler(description)
+			def descMap = zigbee.parseDescriptionAsMap(description)
+			if (descMap?.clusterInt == zigbee.POWER_CONFIGURATION_CLUSTER) {
+				map = batteyHandler(description)
+			} else if (descMap?.clusterInt == zigbee.IAS_ZONE_CLUSTER && descMap.commandInt != 0x07 && descMap.value) {
```

Comment:
This check should probably be 
`descMap?.clusterInt == zigbee.IAS_ZONE_CLUSTER && (descMap.commandInt == 0x01 || descMap.commandInt == 0x0A)` as the following code is only valid if this is a read attr or report attr message.

Reply:
I agree but I know there are a lot of DTHs that do the same thing. I think we can leave it for now and could fix them all up at the same time